dev/helpers/mssql/run_migrations.sh

```diff
@@ -61,18 +72,11 @@ migrate () {
 }
```

Comment:
Remove `echo $file > $LAST_MIGRATION_FILE` from the `migrate ()` function above.